property_tests/tests/init_method/package.json

```diff
@@ -0,0 +1,12 @@
+{
+    "scripts": {
+        "test": "ts-node --transpile-only --ignore=false test/test.ts"
+    },
+    "dependencies": {
+        "azle": "^0.18.5"
```

Comment:
Add to the deployment script to change these versions

Reply:
it is now